public/app/features/explore/RichHistory/RichHistory.tsx

```diff
@@ -2,8 +2,7 @@ import React, { PureComponent } from 'react';
 import { css } from 'emotion';
```

Comment:
Change of SortOrder import as we have moved LogsSortOrder to grafana/data and moved SortOrder for Rich history to app/core/utils/richHistory.